scripts/calculator.js

```diff
@@ -0,0 +1,86 @@
+// CALCULATOR OBJECT
+const calculator = {
+  currentOperand: "",
+  previousOperand: "",
+  operator: null,
+}
+// REFRESH DISPLAY
+function updateDisplay() {
+  document.querySelector("[data-current-operand]").innerText = calculator.currentOperand;
+  document.querySelector("[data-previous-operand]").innerText = calculator.previousOperand;
+}
+//DETECT WHEN CLEAR BUTTON IS PUSHED
+document.querySelector("[data-clear]").addEventListener(
+  "click", clearCalculator)
+//CLEAR CALCULATOR
+function clearCalculator() {
+  calculator.currentOperand = "";
+  calculator.previousOperand = "";
+  calculator.operator = null;
+  updateDisplay()
+}
+//DETECT WHEN CLEAR BUTTON IS PUSHED
+document.querySelector("[data-delete]").addEventListener(
+  "click", deleteCurrentOperand)
+//DELETE LAST INTERGER OF CURRENT OPERAND
+function deleteCurrentOperand() {

```

Comment:
If it's only removing _the terminating character_ of the current operand, then it might be an idea to name the function differently. `deleteCurrentOperand` implies the whole operand to me.